app/workers/follows/send_email_notification_worker.rb

```diff
@@ -7,6 +7,8 @@ def perform(follow_id, mailer = NotifyMailer.name)
       follow = Follow.find_by(id: follow_id, followable_type: "User")
       return unless follow&.followable.present? && follow.followable.receives_follower_email_notifications?
 
+      return if follow.follower.score < 25 # Restrict new follower emails to more active/established accounts
```

Comment:
Instead of having a hard-coded value, do we maybe want to add this to `SiteConfig`?